app/views/transient_registrations/show.html.erb

```diff
@@ -302,6 +243,48 @@
             </tbody>
           </table>
         <% end %>
+
+
+        <% if @transient_registration.main_people.any? %>
+          <h2 class="heading-medium">
+            <%= t(".person_information.heading.#{@transient_registration.business_type}") %>
+          </h2>
+
+          <table>
+            <tbody>
+              <tr>
+                <th>
+                  <%= t(".person_information.name") %>
+                </th>
+                <th>
+                  <%= t(".person_information.dob") %>
+                </th>
+                <th>
+                  <%= t(".person_information.convictions") %>
+                </th>
+              </tr>
+              <% @transient_registration.main_people.each do |person| %>
+                <tr>
+                  <td>
+                    <%= person.first_name %> <%= person.last_name %>
+                  </td>
+                  <td>
+                    <%= person.dob.to_date %>
+                  </td>
+                  <td>
+                    <% if person.conviction_search_result.present? %>
+                      <div class="bold-xsmall">
+                        <%= t(".person_information.has_convictions.yes") %>
+                      </div>
+                    <% else %>
+                      <%= t(".person_information.has_convictions.no") %>
```

Comment:
Why is the `yes` styled to be small and the `no` isn't?

Reply:
is not small, is `bold` :D GDS Elements